snippets/core/system-text-json/csharp/Utf8ReaderFromFile.cs

```diff
@@ -10,7 +10,7 @@ public class Utf8ReaderFromFile
         private static readonly byte[] s_nameUtf8 = Encoding.UTF8.GetBytes("name");
         public static void Run()
         {
-            // Read as UTF-16 and transcode to UTF-8 to convert to a Span<byte>
+            // Read as UTF-16 and transcode to UTF-8 to convert to a ReadOnlySpan<byte>
```

Comment:
Since we already mention it, maybe add the code snippet to strip the UTF-8 BOM.
```C#
public class Utf8ReaderFromFile
{
    private static readonly byte[] s_nameUtf8 = Encoding.UTF8.GetBytes("name");
    private static ReadOnlySpan<byte> Utf8Bom => new byte[] { 0xEF, 0xBB, 0xBF };

    public static void Run()
    {
        // Read as UTF-16 and transcode to UTF-8 to convert to a ReadOnlySpan<byte>
        //string fileName = @"C:\Users\ahkha\Desktop\jsonWithUtf8BOM.json";
        //string jsonString = File.ReadAllText(fileName);
        //ReadOnlySpan<byte> jsonReadOnlySpan = Encoding.UTF8.GetBytes(jsonString);

        // Or ReadAllBytes if the file encoding is UTF-8:
        string fileName = @"C:\Users\ahkha\Desktop\jsonWithUtf8BOM.json";
        ReadOnlySpan<byte> jsonReadOnlySpan = File.ReadAllBytes(fileName);

        // Read past the UTF-8 BOM bytes if one exists.
        if (jsonReadOnlySpan.StartsWith(Utf8Bom))
        {
            jsonReadOnlySpan = jsonReadOnlySpan.Slice(Utf8Bom.Length);
        }

        // Rest of the existing code
   }
}
```